code/modules/antagonists/abductor/machinery/experiment.dm

```diff
@@ -116,52 +116,54 @@
  * * type The type of experiment to be performed
  * * user The mob starting the experiment
  */
-/obj/machinery/abductor/experiment/proc/experiment(mob/occupant, type, mob/user)
+/obj/machinery/abductor/experiment/proc/experiment(mob/living/carbon/human/occupant, type, mob/user)
 	LAZYINITLIST(history)
-	var/mob/living/carbon/human/H = occupant
 
-	if(istype(H)) //We shouldn't be processing anything other than humans, and if we do we runtime.
-		return
+	if(!istype(occupant))
+		stack_trace("Abductor '[name]' called /proc/experiment with unexpected occupant ([occupant])")
+
+	if(!ishuman(occupant)) //We shouldn't be processing anything other than humans
```

Comment:
Isn't this the exact same thing as the istype above it?

Reply:
No, istype checks for any type, ishuman just for human (which can happen without any bugs). The istype checks if it’s not some weird null or nonsensical value and then throws a runtime error. I don’t really see the point but san wanted it